pom.xml

```diff
@@ -98,8 +99,8 @@
             --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
 
             -Dcassandra.config=file://${session.executionRootDirectory}/test/conf/cassandra.yaml
```

Comment:
this file was missing, so I copied one from the C* codebase (you'll see it below)